### DISCUSSION TITLE
Can't open UTM after installation

### DISCUSSION THREAD

**Discussion by eurakaka** [utmapp/UTM#6122] [UTM Help (iOS)]
**Describe the issue**
Can't open UTM after installation  
**Configuration (required)**
* UTM Version: latest(not hv version ), 4.0.0, 3.2.x, 2.2.x etc
* OS Version: ios15.5
* Device Model: ipad8 with A12
* Is it jailbroken (name jailbreak used)? : rootless jail with dopamine, trollstore,sileo,zebra installed
* How did you install UTM? : 1.downloading ipa from github release, installed by trollstore . I've tried sileo or zebra but failed to Add [https://cydia.getutm.app/] for UTM and failed to launch deb on seilo. I‘ve tried install cydia but it requires rootful jail. I am deeply confused and really need help.

**Comment by lurenJBD** [comment]


**Reply by n-WN** [reply]
deb files cannot be installed via TrollStore2
